### DISCUSSION TITLE
Set border colors for specific inactive windows

### DISCUSSION THREAD

**Discussion by crazygolem** [kovidgoyal/kitty#7907] [Q&A] [answered]
It doesn't seem possible to set `inactive_border_color` to a different value for specific windows using `kitten @ set-colors`, it always sets the color for all inactive windows. (For the active border this is not a problem, because there is always only one active window.)

For example:

    kitten @ set-colors -m id:2 inactive_border_color=blue

sets the color of all inactive window borders to blue. While:

    kitten @ set-colors -m id:2 cursor=blue

only sets the cursor's color for the window id:2 to blue, leaving the other windows unchanged.

Is there a way to achieve it in python, e.g. from a watcher, by manipulating `Window` objects instead of using the `set-colors` remote control?

**Comment by kovidgoyal** [comment, marked as answer]
No, border colors are global, not per window.
